src/packages/sdk/src/__tests__/__snapshots__/engineCommands.test.ts.snap

```diff
@@ -3765,7 +3765,7 @@ Object {
               "authorId",
             ],
             "relationName": "PostToUser",
-            "relationOnDelete": "NONE",
+            "relationOnDelete": "Cascade",
```

Comment:
This should be related to https://github.com/prisma/prisma-engines/pull/1947#discussion_r656198290

Not sure about the other changes, could you check @williamluke4 @millsp ?

Reply:
We need to double check why the `relationOnDelete` changed even when the preview feature is not enabled.

Reply:
I think only Amplication is using it in code https://github.com/amplication/prisma-schema-dsl/blob/d7126f3ea580090bb287be34258fc13b00db6149/src/builders.ts#L169-L198